data.sql

```diff
@@ -5,3 +5,13 @@ insert into animals(name,date_of_birth,escape_attempts,neutered,weight_kg) value
 insert into animals(name,date_of_birth,escape_attempts,neutered,weight_kg) values('Pikachu','7-Jan-2021',1,false,15.04);
 insert into animals(name,date_of_birth,escape_attempts,neutered,weight_kg) values('Devimon','12-May-2017',5,true,11);
 
+
+insert into animals(name,date_of_birth,escape_attempts,neutered,weight_kg) values('Charmander','8-Feb-2020',0,false,11);
+insert into animals(name,date_of_birth,escape_attempts,neutered,weight_kg) values('Plantmon','15-Nov-2021',2,true,5.7);
+insert into animals(name,date_of_birth,escape_attempts,neutered,weight_kg) values('Squirtle','2-Apr-1993',3,false,12.13);
+insert into animals(name,date_of_birth,escape_attempts,neutered,weight_kg) values('Angemon','12-Jun-2005',1,true,45);
```

Comment:
- [ ] As indicated by the previous reviewer and as per project requirements, these first four animals have negative signs infront of their weights. Kindly add it.